app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import re
+
+
+def copy_file(command: str):
+    file_names = re.findall(r"(\w+[-_.,]?\w+.txt)", command)
```

Comment:
It is a good solution. But it is not very good to use regex when you can do without it. Refactor.